src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,7 +1,62 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
+import { getMovie } from '../../api';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  addMovie: (movie: Movie) => void;
+};
+
+export const FindMovie: React.FC<Props> = ({ addMovie }) => {
+  const [querySearch, setQuerySearch] = useState('');
+  const [movie, setMovie] = useState<Movie>();
+  const [error, setError] = useState(false);
+  const [isLoading, setLoading] = useState(false);
+
+  const handleClick
+    = async (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+      e.preventDefault();
+      setLoading(true);
+      const result = await getMovie(querySearch);
+
+      setLoading(false);
+
+      if ('Title' in result) {
+        const {
+          Title, Plot, Poster, imdbID,
+        } = result;
```

Comment:
```suggestion
      const response = await getMovie(querySearch);

      setLoading(false);

      if ('Title' in response) {
        const {
          Title, Plot, Poster, imdbID,
        } = result;
```
It is better to name it `response`, not result. Because response can be also an error.
Also I think it is better to check if response has `Error`, not if response doesn't have `Title`. Would be more logical

Reply:
Thanks, done